chartfx-dataset/src/main/java/de/gsi/dataset/serializer/spi/ClassFieldDescription.java

```diff
@@ -593,8 +549,7 @@ protected static DataType dataTypeFomClassType(final Class<?> classType) {
         return DataType.OTHER;
     }
 
-    protected static void exploreClass(final Class<? extends Object> classType, final ClassFieldDescription parent,
-            final int recursionLevel, final boolean fullScan) {
+    protected static void exploreClass(final Class<?> classType, final ClassFieldDescription parent, final int recursionLevel, final boolean fullScan) {
```

Comment:
Codacy found an issue: [The method 'exploreClass(Class, ClassFieldDescription, int, boolean)' has an NPath complexity of 832](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5764587)